posts/saving-and-analyzing-trending-topics-on-twitter-using-aws-athena-lambda-and-cdk.md

```diff
@@ -0,0 +1,198 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- Twitter
+- AWS
+- AWS Lambda
+- AWS Athena
+- AWS CDK
+date: 2020-08-03T14:50:55.000Z
+title: "Saving and analyzing trending topics on Twitter using AWS Athena, Lambda and CDK"
+image:
+---
+
+With more than 300 million active users, Twitter is still the best platform to provide efficient and accurate real time news. The SMS of the Internet is also one of the best social network and is particularly used a lot by all kind of developers which makes it a very good source of tech information.
```

Comment:
The SMS of the Internet is also one of the best social networks and is used by a plethora of developers, which makes it a prime source of tech information.